sorting/quicksort3.cpp

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+#include <vector>
+#include <cstdlib>
+
+using std::vector;
```

Comment:
Don't use `using` directive it's not good practice.Please remove `using ` and directly use `std::` wherever necessary.